packages/nitro-protocol/hardhat.config.ts

```diff
@@ -19,7 +19,11 @@ module.exports = {
   solidity: '0.6.12',
   namedAccounts: {
     deployer: {
-      rinkeby: 0,
+      // Address used for the initial deploy of the contracts.
+      // Ask @kerzhner for the private key if you would like to deploy from the same address
+      // Otherwise uncomment the line below
+      //rinkeby: 0,
+      rinkeby: '0x87612aAD373586A38062c29F833A2AbC72038591',
```

Comment:
I guess this is also a `build: ` commit, since you're not fixing any code?

Reply:
`build` is probably a better characterization.